LibsAndSamples.sln

```diff
@@ -234,6 +234,10 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WPF", "tests\devapps\WAM\MS
 EndProject
 Project("{C7167F0D-BC9F-4E6E-AFE1-012C56B48DB5}") = "WPF.Package", "tests\devapps\WAM\MSIX\WPF.Package\WPF.Package.wapproj", "{34E323E8-E706-4DF4-B916-D614FC8CFCEB}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NetCoreConsoleWAM", "tests\devapps\WAM\NetCoreConsoleWAM\NetCoreConsoleWAM.csproj", "{7CA7CBEA-8F8E-48EF-8003-AB645B9DBD75}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NetFxConsoleWAM", "tests\devapps\WAM\NetFxConsoleWAM\NetFxConsoleWAM.csproj", "{5CBA39DF-AE9A-449A-808C-57476B78A7F6}"
+EndProject

```

Comment:
Can these two projects be combined? Just multi-target?

Reply:
We can I guess but leaving it as is for now